Auth0Tests/Responses.swift

```diff
@@ -28,34 +27,57 @@ let RecoveryCode = "162534"
 let MFAToken = UUID().uuidString.replacingOccurrences(of: "-", with: "")
 let AuthenticatorId = UUID().uuidString.replacingOccurrences(of: "-", with: "")
 let ChallengeTypes = ["oob", "otp"]
-let APISuccessStatusCode = Int32(200)
+let APISuccessStatusCode = 200
 let APIResponseHeaders = ["Content-Type": "application/json"]
 
-func catchAllResponse() -> HTTPStubsResponse {
-    return HTTPStubsResponse(error: NSError(domain: "com.auth0", code: -99999, userInfo: nil))
+func catchAllResponse() -> RequestResponse {
+    return { request in
+        let error = NSError(domain: "com.auth0", code: -99999, userInfo: nil)
+        return (nil, nil, error)
+    }
 }
 
-func apiSuccessResponse(json: [AnyHashable: Any] = [:]) -> HTTPStubsResponse {
-    return HTTPStubsResponse(jsonObject: json, statusCode: APISuccessStatusCode, headers: APIResponseHeaders)
+func apiSuccessResponse(json: [AnyHashable: Any] = [:]) -> RequestResponse {
+    return { request in
+        let data = try! JSONSerialization.data(withJSONObject: json, options: [])
+        let response = HTTPURLResponse(url: request.url!, statusCode: APISuccessStatusCode, httpVersion: nil, headerFields: APIResponseHeaders)
```

Comment:
Should we force-unwrap the response here as well?

Reply:
Yes, missed it, added it now